src/content/test/pointer-motion/pointer-gestures.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { create, React } from '../../common';
+
+export const infoAndExamples = create(({ Markup }) => (
+    <>
+        <h1>Pointer gestures</h1>
+        <p>Functions must be operable without requiring multipoint or path-based gestures.</p>
+
+        <h2>Why it matters</h2>
+        <p>
+            Multipoint and path-based gestures can be difficult or impossible to execute for people who have limited fine motor control or
+            who use head tracking, gaze tracking, or speech commands to control their mouse pointer. Moreover, many users might not be aware
+            that such gestures are supported, especially if they have cognitive or learning disabilities.
+        </p>
+
+        <h2>How to fix</h2>
+        <p>
+            For any function that can be operated using multipoint or path-based gestures, provide an alternative method that (1) requires
+            only a single pointer, and (2) doesn't depend on the path of the pointer's movement.
+        </p>
+
+        <h2>Example</h2>
+        <Markup.PassFail
+            failText={
+                <p>
+                    An online survey requires respondents to rank the importance of certain list items by dragging and dropping them into
```

Comment:
As I was reading this, it honestly hadn't occurred to me that a drag-and-drop interaction like this would count as a "path-based gesture"; I had been imagining examples like "a finger painting web app". I wonder if this was just my bias showing or if drag-and-drop is a sufficiently frequent offender of this case that it might be worth including higher up in the page (ie, "multipoint, path-based, or drag-and-drop gestures")

Reply:
(maybe I'm overthinking it and this is just feedback that this is a very valuable example :))